### ISSUE TITLE
[NETSDKE2E] [Insiders] Asian language localization display anomalies when using tar.gz to install VS Code.

### ISSUE THREAD

**Issue by yinjiehe01** [microsoft/vscode#232240]
Affected Build:
VS Code 1.95.0-insider
Repro OS: Linux

Steps to Reproduce:

1. Use the tar.gz to install VS Code insiders from https://code.visualstudio.com/insiders/#linux in Linux
2. Change the VS Code language to Asian language like Simple Chinese, Japanese
3. Obverse.

Expected Result:
Asian languages ​​need to be fully localized.
SC:
![Image](https://github.com/user-attachments/assets/96cb2d58-3520-4062-9726-a36e1e74b256)
JP:
![Image](https://github.com/user-attachments/assets/ef260aed-4d35-4c90-9021-aa16b27ff410)
Actual Result:
Asian language localization display anomalies.
SC:
![Image](https://github.com/user-attachments/assets/3cb89083-db26-4ccc-a77e-494e883d7719)
JP:
![Image](https://github.com/user-attachments/assets/fd304a25-115d-46a8-a6d5-016e244ab09f)

**Comment by TylerLeonhardt** [comment]
![Image](https://github.com/user-attachments/assets/041f126d-e21c-4227-b41a-5990b7cccf11)

I just tried this on Debian 12 and it looks ok.

* Is this still reproing for you with the latest VS Code Insiders?
* What OS are you on? What Linux Window Manager are you using?

**Comment by yinjiehe01** [comment]
I used SLESE 15.6 with VS Code Inisders and it still repro.

**Comment by deepak1556** [comment]
Given the issue affects native titlebar as well, wondering if its a gpu rendering issue. Can you start the application with `code-insiders --disable-gpu` and confirm if the issue persists ?

**Comment by yinjiehe01** [comment]
![Image](https://github.com/user-attachments/assets/cdd73a15-36af-4daf-9984-a5a597e024a8)
it still repro's by using command code-insiders --disable-gpu

**Comment by TylerLeonhardt** [comment]
I'm not 100% certain that `--disable-gpu` is applied on that restart. Can you double check by running it manually instead of having VS Code restart itself because you changed the language?

**Comment by deepak1556** [comment]
Yeah explicit flags wouldn't persist from in-app restarts. Please try the following setting which will persist on restart https://code.visualstudio.com/updates/v1_40#_disable-gpu-acceleration

**Comment by yinjiehe01** [comment]
![Image](https://github.com/user-attachments/assets/3d24179b-aecf-4379-9c49-70476c6477bb)
I have tried the setting and restart VS Code again by using command. it still no effect.

**Comment by deepak1556** [comment]
Thanks for checking, to reconfirm are you seeing the localization issues only with the titlebar or even in the editor area. It is kind of hard to tell from the screenshot.

If it is only in the titlebar area can you switch to custom titlebar `"window.titleBarStyle": "custom"` and see how that behaves.

**Comment by yinjiehe01** [comment]
![Image](https://github.com/user-attachments/assets/2b9acc93-b49a-4245-8611-fb5d1dc8b23a)
The localization issues are not only in the title but also in the edit page

**Comment by deepak1556** [comment]
Can you launch with `--verbose --locale="zh-cn"` flag and attach the output

**Comment by deepak1556** [comment]
https://github.com/microsoft/vscode/issues/232240#issuecomment-2446170368 is not meant to address the issue, I am interested in the logs to see if there is some related error from the runtime. Can you please rerun the command and share the output logs.

**Comment by yinjiehe01** [comment]
here is the output logs.[logs.txt](https://github.com/user-attachments/files/17580728/logs.txt)

**Comment by TylerLeonhardt** [comment]
This is on OpenSUSE Leap 15.6, a bit closer to what you're using, and it looks ok as well:
![Image](https://github.com/user-attachments/assets/c7907519-d3fb-47ec-b9e3-9adf4f1db8d7)

Looking at your screenshot, it appears that the correct translations are there... (`File` appears to be right, and the last one, which I think is `Help`, has only the 2nd character) it's just some of the characters are missing... so I think we can rule out that this is a Language Pack install issue...

It does feel like this is some sort of a rendering issue with the runtime or maybe the font... @deepak1556 maybe you have other ideas to narrow it down.

**Comment by deepak1556** [comment]
To confirm, do you see the issue when running the deb or rpm package of VSCode ?

**Comment by yinjiehe01** [comment]
I had no problem when running the deb or rpm package of VS Code.

**Comment by yinjiehe01** [comment]
confirmed the issue does not repro in VS Code version 1.100
![Image](https://github.com/user-attachments/assets/1aeb969d-395d-4ecb-a7dd-85dd420b073f)
